packages/calcite-components/src/components/list-item-group/list-item-group.scss

```diff
@@ -16,10 +16,16 @@
 }
 
 ::slotted(calcite-list-item) {
-  @apply shadow-border-bottom mb-px;
+  @apply shadow-border-top;
+  margin-block-start: 1px;
 }
 
-::slotted(calcite-list-item:last-child) {
+::slotted(calcite-list-item:first-of-type) {
+  @apply shadow-none;
+}
+
+// removes shadow for the first item in filteredItems of the group.
```

Comment:
Not sure if the comment or the selector needs updating, but this would affect both filtered and unfiltered items, no?

Reply:
Yup it affects both filtered and unfiltered items. will update the comment and remove the selector above `::slotted(calcite-list-item:first-of-type)` which is redundant.